src/plugins/intel_gpu/src/graph/include/convolution_inst.h

```diff
@@ -107,6 +107,15 @@ struct typed_program_node<convolution> : public typed_program_node_base<convolut
         return params;
     }
 
+    size_t hash() const override {
+        if (!seed) {
+            seed = parent::hash();
+
+            seed = hash_combine(seed, transposed);
+        }
+        return seed;
```

Comment:
Can this be fixed to use primitive::hash() && hash_combine(seed, transposed) ?

Reply:
I think it is hard to change it. because primitive::hash() is used only in primitive but this is convolution_node which is not primitive. 
If primitive::hash() is used instead of parent::hash(), we should add codes to calculate hash of activation and fused params in convolution_node::hash().